src/types/Todo.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Todo: Represents a task that needs to be done
+ */
```

Comment:
I've realized that VS Code will show this when hovering over a Type/Function elsewhere in the codebase, very nice.

- [ ] Add a comment like this to all types and modules